Cargo.toml

```diff
@@ -25,7 +25,7 @@ targets = [
 ]
 
 [dependencies]
-libc = { version = "0.2.102", features = [ "extra_traits" ] }
+libc = { git = "https://github.com/rust-lang/libc", rev = "971fde3d7", features = [ "extra_traits" ] }
```

Comment:
I'm fine waiting until a new libc release is cut for this to be merged. No urgency on my end.